src/app/components/Sidebar/index.tsx

```diff
@@ -157,6 +166,11 @@ const SidebarFooter = (props: SidebarFooterProps) => {
           </Box>
         </Menu>
       </Box>
+      <Box align="center" pad="small">
+        <a href="https://github.com/oasisprotocol/oasis-wallet-web" target="_blank" rel="noopener">
```

Comment:
We already have two links to source :D  
![image](https://user-images.githubusercontent.com/3758846/159960546-b9a8fd60-dadf-41b8-9ef6-d9b9dc615c57.png)

Reply:
lol I will validate this task with Tadej now :D

Reply:
so the answer is "Just have both for the time being".

Reply:
okay :shrug: